src/Microsoft.Azure.SignalR.Protocols/GroupMessage.cs

```diff
@@ -247,45 +247,4 @@ public LeaveGroupWithAckMessage(string connectionId, string groupName, int ackId
             TracingId = tracingId;
         }
     }
-
-    /// <summary>
-    /// A waiting for ack check-user-in-group message.
-    /// </summary>
-    public class CheckUserInGroupWithAckMessage : ExtensibleServiceMessage, IAckableMessage, IMessageWithTracingId
```

Comment:
Breaking change?

Reply:
why? think the message instance <-> bytes is enough

Reply:
As this is a public class, users might already depend on the class.

Reply:
i do not think it is a breaking change.